app/extensions/email/__init__.py

```diff
@@ -126,7 +128,8 @@ def __init__(self, *args, **kwargs):
 
         # will attempt to discover via set_language() unless specifically set
         self.language = None
-        self._original_recipients = None
+        self._original_recipients = None  # should only be set by resolve_recipients
+        self.recipients = kwargs['recipients']
```

Comment:
Not a huge issue but this will be overwritten by the Message constructor so isn't necessary. Also if the message constructor didn't overwrite it, then the override_recipients bit below wouldn't work.